src/router/index.js

```diff
@@ -1,3 +1,7 @@
-const routesList = []
+import { Router } from 'express'
+
+const routes = Router()
```

Comment:
You forgot to initiate the Router instance. Here it should be:
const routes = new Router()